src/fileheader.h

```diff
@@ -42,6 +42,7 @@ namespace zim
       static const uint16_t zimOldMajorVersion;
       static const uint16_t zimMajorVersion;
       static const uint16_t zimMinorVersion;
+      static const uint16_t zimMinorVersionWithoutV0;
```

Comment:
This remains unused

Reply:
Done (the order of the commit has bit changed as this commit use zimMinorVersionWithoutV0 and its used in removed in next commits. New order introduce commit bumping minor version after commit not loading v0)